setup.cfg

```diff
@@ -0,0 +1,76 @@
+[metadata]
+name = asdf_astropy
+author = The ASDF dev team
+author_email = 
+license = BSD 3-Clause
+license_file = licenses/LICENSE.rst
+url = http://docs/astropy.org/projects/asdf-astropy
+description = Astropy Converters for ASDF
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+edit_on_github = False
+github_project = astropy/asdf-astropy
+
+[options]
+zip_safe = False
+packages = find:
+python_requires = >=3.6
+setup_requires = setuptools_scm
+install_requires =
+    astropy
+    asdf
+    numpy
+    asdf-transform-schemas
+
+[options.entry_points]
+asdf_extensions =
```

Comment:
The name of the entry point is different for the new-style plugin.  This should be:

```
asdf.converter_providers =
   asdf-astropy = asdf_astropy.extension:TransformConverterProvider
```